src/assets/styles/index.scss

```diff
@@ -7,7 +7,8 @@
 @import "~styles/header";
 @import "~styles/event_list";
 @import "~styles/event_card";
-@import "~styles/pikaday";
+@import "~styles/vendor/pikaday";
+@import "~styles/vendor/mapbox-gl.css";
```

Comment:
IMO importing from node_modules is probably ok.....tim is doing that over here for the date picker as well, to import base pikaday styles then override: https://github.com/aclu-people-power/map/pull/69/files#diff-aeb7f7b7447880522fbc3ecbe64310daR1

We could probably do something fancier with the webpack config too but IMO it would probably be about the same end result :man_shrugging: